revent/settings.py

```diff
@@ -142,12 +142,12 @@
 
 # Django-storages
 
-STORAGES = {
-    "default": {
-        "BACKEND": "storages.backends.s3.S3Storage",
-        "OPTIONS": {
-            "bucket_name": AWS_STORAGE_BUCKET_NAME,
-            "location": AWS_S3_ENDPOINT_URL,
-        },
-    },
-}
+# STORAGES = {
```

Comment:
What was the error? Was it a 403 error?
If so this [documentation](https://django-storages.readthedocs.io/en/1.5.2/backends/amazon-S3.html) might be helpful

Reply:
<img width="1283" alt="Screenshot 2023-09-26 at 09 37 05" src="https://github.com/runtimerevolution/revent-api/assets/88937009/f06a1906-b25a-4eac-a12e-e7b0f8a57564">

this is the issue I'm having

Reply:
it was fixed, check the settings.py file